caching/caching.go

```diff
@@ -103,10 +102,8 @@ func (c *Caching) Initialize() {
 		time.Sleep(time.Duration(float64(c.cachingInterval) * rand.Float64()))
 		ticker := time.NewTicker(c.cachingInterval)
 		for {
```

Comment:
`for range ticker.C { ... }` would be the idiomatic approach.

Reply:
Thanks @dominikh !